lib/mode.coffee

```diff
@@ -0,0 +1,234 @@
+XRegExp = require 'xregexp'
+sb_exec = require 'sb-exec'
+path = require 'path'
+
+pattern = XRegExp('(?<file>[^\n\r]+):(?<from_line>\\d+):(?<from_col>\\d+):\\s*\
+  (?<to_line>\\d+):(?<to_col>\\d+)\\s+\
+  ((?<error>error|fatal error)|(?<warning>warning)|(?<info>note|help)):\\s+\
+  (?<message>.+?)[\n\r]+($|(?=[^\n\r]+:\\d+))', 's')
+
+parseOldMessages = (output, disabledWarnings) ->
+  elements = []
+  XRegExp.forEach output, pattern, (match) ->
+    if match.from_col == match.to_col
+      match.to_col = parseInt(match.to_col) + 1
```

Comment:
Correct, `atom-linter::exec` is a wrapper for `sb-exec` so there is no need to include it directly.